config/quality/pmd/pmd-ruleset.xml

```diff
@@ -11,7 +11,7 @@
 
     <rule ref="rulesets/internal/all-java.xml/TooManyStaticImports">
         <properties>
-            <property name="legalPackages" type="String"
```

Comment:
I removed it because Android Studio says it's not allowed there. But this change isn't really relevant to the rest of the PR, so I just removed this.